src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,24 @@
-export const TodoInfo = () => {};
+import cn from 'classnames';
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo,
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  return (
```

Comment:
better to move logic with the user here
<img width="423" alt="image" src="https://github.com/mate-academy/react_add-todo-form/assets/70220667/bdd20259-fb3c-4b36-ad78-1668ece8ec02">

and if the user doesn't exist then do not render the UserInfo